pkg/osquery/tables/airport/table_darwin_test.go

```diff
@@ -0,0 +1,195 @@
+//go:build darwin
+// +build darwin
+
+package airport
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	"github.com/osquery/osquery-go/plugin/table"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_parseAirportOutput_HappyPath(t *testing.T) {
+	t.Parallel()
+
+	type args struct {
+		option       string
+		queryContext table.QueryContext
+	}
+	tests := []struct {
+		name      string
+		args      args
+		assertion assert.ErrorAssertionFunc
+	}{
+		{
+			name: "scan",
+			args: args{
+				option:       "scan",
+				queryContext: table.QueryContext{},
+			},
+			assertion: assert.NoError,
+		},
+		{
+			name: "getinfo",
+			args: args{
+				option:       "getinfo",
+				queryContext: table.QueryContext{},
+			},
+			assertion: assert.NoError,
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+
+			inputFile, err := os.Open(fmt.Sprintf("testdata/%s.input.txt", tt.name))
+			assert.NoError(t, err)
+			defer inputFile.Close()
+
+			input, err := ioutil.ReadAll(inputFile)
+			assert.NoError(t, err)
+
+			got, err := processAirportOutput(bytes.NewReader(input), tt.args.option, tt.args.queryContext, log.NewNopLogger())
+			tt.assertion(t, err)
+
+			wantFile, err := os.Open(fmt.Sprintf("testdata/%s.output.json", tt.name))
+			assert.NoError(t, err)
+			defer wantFile.Close()
+
+			wantBytes, err := ioutil.ReadAll(wantFile)
+			assert.NoError(t, err)
+
+			var want []map[string]string
+			err = json.Unmarshal(wantBytes, &want)
+			assert.NoError(t, err)
+
+			assert.ElementsMatch(t, want, got)
+		})
+	}
+}
+
+func Test_parseAirportOutput_EdgeCases(t *testing.T) {
+	t.Parallel()
+
+	type args struct {
+		output       io.Reader
+		option       string
+		queryContext table.QueryContext
+		logger       log.Logger
```

Comment:
I'd probably drop this from the test args, and pass `log.NewNopLogger()` directly in t.Run